packages/plugin-rsc/src/plugin.ts

```diff
@@ -1256,6 +1263,12 @@ function vitePluginUseClient(
               const meta = manager.clientReferenceMetaMap[id]
               if (meta) {
                 meta.renderedExports = mod.renderedExports
+                meta.serverChunk = normalizePath(
+                  path.relative(
+                    manager.config.root,
+                    chunk.facadeModuleId ?? chunk.fileName,
+                  ),
+                )
```

Comment:
Not sure exactly how and when `facedeModuleId` becomes empty, but that seems to be the case for one of chunk on https://github.com/wakujs/waku/pull/1653